releases/release-1.32/release-notes/release-notes-draft.md

```diff
@@ -2,13 +2,6 @@
 
 ### (No, really, you MUST read this before you upgrade)
```

Comment:
Should we add https://github.com/kubernetes/kubernetes/issues/129009#issuecomment-2518321729 here? @msau42 is this an upgrade warning?

Reply:
IMHO: not an upgrade warning, CSI driver developers's _own_ binaries may or may not switch to go 1.23 and that's not part of upgrading a cluster. This is comparable to the scheduler plugin changes. (ref https://github.com/kubernetes/kubernetes/pull/129126)

Reply:
IE: these are warnings for third party tools to consider, as a user upgrading a cluster you should wait until your drivers, custom schedulers, etc advertise 1.32 compatibility. As part of those tools implementing 1.32, they should consider these warnings. Versus something like a breaking change you must handle in one of the core components (like removal of dockershim).

Reply:
We've decided on not adding it as a Known Issue https://kubernetes.slack.com/archives/C2C40FMNF/p1733579835815919?thread_ts=1733471881.209099&channel=C2C40FMNF&message_ts=1733579835.815919 .